WeatherMan.py

```diff
@@ -0,0 +1,33 @@
+import argparse
+import sys
+
+from DataReader import data_reader
+from View import View
+
+weather_report = {}
+
+data_reader = data_reader()
+view = View()
+
+arg_parser = argparse.ArgumentParser()
+arg_parser.add_argument("[report#]", type = int, default = 0, help = 'report number')
+arg_parser.add_argument("[data_dir]", type = str, default = "none", help = 'data directory')
+arguments = arg_parser.parse_args()
+
+if getattr(arguments, "[report#]") == 1:
+	annual_report = data_reader.generate_annual_report(getattr(arguments, "[data_dir]"))
+	if annual_report:
+		view.print_annual_report(annual_report)
+
+elif getattr(arguments, "[report#]") == 2:
+	annual_report = data_reader.generate_max_temp_report(getattr(arguments, "[data_dir]"))
+	if annual_report:
+		view.print_annual_max_report(annual_report)
+
+else:
+	print 'Usage: WeatherMan [report#] [data_dir]\n'
```

Comment:
Check the input by using `argparser`. `argparser` will print this when input is not valid and when `--help` is given.
**Hint:** `argparser` prints help `params` of arguments when invoked with incorrect values.